flask_rest_jsonapi/api.py

```diff
@@ -20,6 +21,10 @@ def __init__(self, app=None, blueprint=None, decorators=None):
         :param blueprint: a flask blueprint
         :param tuple decorators: a tuple of decorators plugged to each resource methods
         """
+
+        if app is None:
+            app = Flask(__name__)
```

Comment:
I dont think this is a good idea:

If the user create a instance of the jsonapi, then configure it using .init_app(app):
the user would have created a second Flask app, 

instead, __init__ shoud try to call init_app if app is not None.